tests/test_rsa.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+# First-party
+from todo_project_name import rsa
+
+# Third-party
+import pytest
+
+
+def test_rsa_file_operations(tmp_path):
+    KEY_TYPE = rsa.RSAKeyPublic
+    key_types = set([rsa.RSAKeyPublic, rsa.RSAKeyPrivate])
```

Comment:
Why not `{ rsa.RSAKeyPublic, rsa.RSAKeyPrivate }`?

Reply:
Right, thanks! I started with `set()` and reacted to the message about iterable.